### PR TITLE
Add California Clean Vehicle Rebate Project

### PR DESCRIPTION
**Pull request by MaxGhenis**
# New variable: `ca_cvrp`

- [x] Label field added
- [x] Documentation field added
- [x] Unit field added
- [x] Default value field added if relevant
- [x] Variable name follows conventions
- [x] Unit test(s) added
- [x] Integration test(s) added if relevant
- [x] Issues this PR fixes linked

### CODE REVIEW

File: openfisca_us/variables/states/ca/calepa/carb/cvrp/ca_cvrp_vehicle_rebate_amount.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from openfisca_us.model_api import *
+
+
+class ca_cvrp_vehicle_rebate_amount(Variable):
+    value_type = float
+    entity = Person
+    definition_period = YEAR
+    label = "CVRP rebate for purchased vehicle"
+    unit = USD
+    documentation = "Rebate value for a purchased vehicle under the California Clean Vehicle Rebate Project (CVRP)"
```
Comment (nikhilwoodruff) [inline openfisca_us/variables/states/ca/calepa/carb/cvrp/ca_cvrp_vehicle_rebate_amount.py:10]: Could add a `reference` attribute to this and other formulas?

File: openfisca_us/variables/states/ca/calepa/carb/cvrp/is_ca_cvrp_increased_rebate_eligible.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    definition_period = YEAR
+    label = "Eligible for CVRP increased rebate"
+    documentation = "Eligible for increased rebate for low- and middle-income participants in California's Clean Vehicle Rebate Project (CVRP)"
+
+    def formula(person, period, parameters):
+        # Use school meal FPG ratio.
+        fpg_ratio = person.spm_unit("school_meal_fpg_ratio", period)
+        p = parameters(period).states.ca.calepa.carb.cvrp.increased_rebate
+        income_eligible = fpg_ratio <= p.fpl_limit
+        categorically_eligible = np.any(
```
Comment (nikhilwoodruff) [inline openfisca_us/variables/states/ca/calepa/carb/cvrp/is_ca_cvrp_increased_rebate_eligible.py:16]: Yeah we should add a pattern for this in `openfisca-tools`.

File: setup.py [hunk 3]
```diff
@@ -4,7 +4,11 @@
 
 setup(
     name="OpenFisca-US",
+<<<<<<< HEAD
```
Comment (nikhilwoodruff) [inline setup.py:7]: Think this is a merge conflict?

File: openfisca_us/variables/states/ca/calepa/carb/cvrp/is_ca_cvrp_increased_rebate_eligible.py (hunk 2 above)
Comment (MaxGhenis) [reply openfisca_us/variables/states/ca/calepa/carb/cvrp/is_ca_cvrp_increased_rebate_eligible.py:17]: Filed https://github.com/PolicyEngine/openfisca-tools/issues/22

**Review by MaxGhenis — COMMENTED** [review]
Changes done

**Review by nikhilwoodruff — APPROVED** [review]

**Pull request merged**